maintainers/maintainer-list.nix

```diff
@@ -10516,6 +10516,12 @@
     githubId = 72201;
     name = "Ole Jørgen Brønner";
   };
+  oliviermarty = {
```

Comment:
This must be done in a separate commit, as per [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md)

> When adding yourself as maintainer, in the same pull request, make a separate commit with the message `maintainers: add <handle>`. Add the commit before those making changes to the package or module

Reply:
Thank you! I fixed the PR.